test/fakeintake/client/client.go

```diff
@@ -55,6 +55,16 @@ import (
 	"github.com/DataDog/datadog-agent/test/fakeintake/client/flare"
 )
 
+const (
+	metricsEndpoint     = "/api/v2/series"
+	checkRunsEndpoint   = "/api/v1/check_run"
+	logsEndpoint        = "/api/v2/logs"
+	connectionsEndpoint = "/api/v1/connections"
+	processesEndpoint   = "/api/v1/collector"
+	containersEndpoint  = "/api/v1/container"
+	flareEndpoint       = "/support/flare"
+)
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do look for something to sincerely praise. -->
**👏 praise**